src/ops/maths.c

```diff
@@ -406,15 +406,17 @@ static int16_t quantize_to_bitmask_scale(int16_t scale_bits, int16_t transpose,
     // accepts 12-bit scale mask and a pitch voltage. transpose is voltage for
     // scale offset. returns nearest pitch voltage in scale.
     if (scale_bits == 0) { return v_in; }  // no active scale bits
-    int16_t sign = (v_in < 0) ? -1 : 1;
-    v_in = (v_in < 0) ? -v_in : v_in;
-    transpose = transpose % table_n[12];
-
-    if (v_in >= table_n[127]) { return table_n[127] * sign; }
-
+	
```

Comment:
the formatting looks off (looks like you have tab set to 2 spaces?)

Reply:
Sure, let me fix up the changelog. For the formatting, I've never been able to run clang format because it's not in @Dewb 's Docker (and I am a Windows person of little Linux talent). I'm not seeing this issue in my raw file, though. It's either got proper tabs (like line 413) or four spaces (like line 415)...

Reply:
does it not show up misaligned for you when you view it on github? this is what i'm seeing:

![image](https://user-images.githubusercontent.com/6855929/157161519-8e5adc35-d667-4215-84f0-abbe5de7028b.png)

Reply:
Hmm, that dependency isn't documented anywhere, good catch!

You can install clang-format from the container's bash prompt by running:
```bash
apt update
apt install clang-format
```

I'll update the docker image to include this.

In the version of clang-format the instructions above will currently install, the command to invoke it is now `git clang-format-6.0`, so unfortunately `make format` won't work.

Reply:
Just pushed a new image, it's based on Ubuntu 20 now and installs a newer version of clang-format so `make format` will indeed work.